README.md

```diff
@@ -1,53 +1,118 @@
 # Documentation
 
-This is where we host all MOC documentation.
+This is the [Mass Open Cloud][moc] documentation repository.
 
-## Contributing
+[moc]: https://massopen.cloud
 
-Documentation files are markdown and are stored in docs/source
+## Organzation
 
-docs/source/home.md is the index for MOC documentation.
+Documentation files are written in Markdown and are stored in the [`docs/source`](docs/source) directory.
 
-For each section in home.md there's a sub directory in docs/source where files relevant to that section are stored. For example files in "How Tos" section are stored in docs/source/how-tos.
+### Adding a new section
 
-We also have an section for archived files. These are stored in docs/source/archive-files and are listed in docs/source/Archives-page.md
+Major sections are defined in `docs/source/index.rst` in the `toctree` block:
 
-Non-markdown files are stored in `docs/souce/_static/<type>/` where sub directory type depends on the file's type. For example all limages are in `docs/source/_static/img`.
```

Comment:
images instead of limages

Reply:
That is deleted text.

Reply:
If you're not familiar with reading diffs, you can view the rendered file by selecting the "..." menu at the upper right of the file and then selecting "View file".

Reply:
thank you, my eye gravitated to the spelling, not the red vs green :)